### PR TITLE
Gen glossary

### PR DESCRIPTION
**Pull request by flossypurse**
**PREVIEW**: https://deploy-preview-1534--mystifying-fermi-1bc096.netlify.app/glossary

### CODE REVIEW

File: docs/glossary.md [hunk 1]
```diff
@@ -10,45 +10,69 @@ tags:
...
+- [Memo](/concepts/what-is-a-memo)
 - [Multi-Cluster Replication](/concepts/what-is-multi-cluster-replication)
 - [Namespace](/concepts/what-is-a-namespace)
 - [Parent Close Policy](/concepts/what-is-a-parent-close-policy)
 - [Query](/concepts/what-is-a-query)
 - [Retention Period](/concepts/what-is-a-retention-period)
 - [Retry Policy](/concepts/what-is-a-retry-policy)
 - [Run Id](/concepts/what-is-a-run-id)
+- [Schedule](/concepts/what-is-a-schedule)
+- [Schedule](/concepts/what-is-a-schedule)
```
Comment (djmagee) [inline docs/glossary.md:46]:
```suggestion
- [Schedule](/concepts/what-is-a-schedule)
```

File: docs/glossary.md [hunk 2]
```diff
@@ -10,45 +10,69 @@ tags:
 The following terms are used in [Temporal Platform](/concepts/what-is-the-temporal-platform) documentation.
 
 - [Activity](/concepts/what-is-an-activity)
+- [Activity Definition](/concepts/what-is-an-activity-definition)
 - [Activity Execution](/concepts/what-is-an-activity-execution)
-- [Activity Heartbeat](/concepts/what-is-an-activity-heartbeat)
+- [Activity Heartbeats](/concepts/what-is-an-activity-heartbeat)
```
Comment (djmagee) [inline docs/glossary.md:15]: Only four terms in this list—Activity Heartbeats, Cron Jobs, Local Activities, and Signals—are plural; the rest are singular. Should those four terms also appear as singular?

**Review by djmagee — COMMENTED** [review]
This is freakin' cool.

File: docs/glossary.md (hunk 2 above)
Comment (flossypurse) [reply docs/glossary.md:15]: Yes, I believe so -

**Review by djmagee — APPROVED** [review]

**Pull request merged**